e2e/cypress/integration/channel/message_reply_pretext_spec.js

```diff
@@ -0,0 +1,80 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+// ***************************************************************
+// - [#] indicates a test step (e.g. # Go to a page)
+// - [*] indicates an assertion (e.g. * Check the title)
+// - Use element ID when selecting an element. Create one if none.
+// ***************************************************************
+
+import users from '../../fixtures/users.json';
+
+const sysadmin = users.sysadmin;
+
+describe('Message Reply with attachment pretext', () => {
+    let newChannel;
+
+    before(() => {
+        // # Enable bot creation
+        cy.apiUpdateConfig({ EnableBotAccountCreation: { Enable: true, }, });
```

Comment:
Here's a more complete command, try this

        cy.apiUpdateConfig({
            ServiceSettings: {
                EnableBotAccountCreation: true,
            },
        });

Reply:
You can check this page locally (login as sysadmin, password is in fixtures/users.json) and see if Enable Bot Account Creation field is set to true

http://localhost:8065/admin_console/integrations/bot_accounts